libs/wingsdk/src/target-tf-azure/bucket.ts

```diff
@@ -92,8 +93,23 @@ export class Bucket extends cloud.BucketBase {
       storageAccountName: this.storageAccount.name,
       containerAccessType: this.public ? "public" : "private",
     });
+  }
+
+  public addObject(key: string, body: string): void {
+    // Blob naming conventions:
+    // https://learn.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#blob-names
+    const blobName = key
+      .split("/")
+      .map((s) => encodeURIComponent(s))
+      .join("/");
 
-    this.storageContainer;
+    new StorageBlob(this, `Blob-${key}`, {
```

Comment:
I don't know much about CdkTF, but I imagine there are limits on the length of the logical id string

Reply:
No restrictions that I'm aware of